guidelines/static/workflow_examples/scenario1copy.json

```diff
@@ -0,0 +1,245 @@
+{
+  "workflow": {
+    "name": "my-workflow",
+    "description": "my awesome workflow",
+    "version": 1,
+    "informatics-gateway": {
+      "ae-title": "MYAET",
+      "data-origins": [
+        {
+          "name": "MY_SCANNER"
+        }
+      ],
+      "export-destinations": [
+        {
+          "name": "PROD_PACS"
```

Comment:
Assuming this is a DICOM-SCU destination, will the user be setting up an SCU destination with the name `PROD_PACS` before registering this workflow? 

If we want the export agents to pick up data from the Workflow Manager, we will need additional arguments. 
For example, the `name` field here can be a unique identifier for the export agent plus additional parameters for the agent to know which SCU to export to.

Reply:
This is an identifier to an SCU that was pre-registered with the IG. It's essentially (in our old terms) the export bucket name. Whoever subscribes to export notifications can subscribe to notifications for any SCU that's pre-registered in the IG.

The actual SCU parameters (url, port, etc) are stored in the IG, and used after the export event is picked up.

Reply:
In that case, if there are 10 SCU registered on IG, then, IG would have to query for all export tasks assigned to those 10 SCUs.  Either an API that takes all 10 SCUs as params or 10 separate calls.

Just to share, in Clara Deploy, we make 2 separate calls, 1 for DICOM and another for DICOMweb.  The SCU parameters are stored in the pipeline definition.

Reply:
Yes, I think we should clarify this (I also commented on the MIG PR regarding exports).

The way I saw it working is that the MWM publishes an event to the Pubsub service with an export notification - that notification contains data about the output data (a reference to the storage layer), as well as the SCU - the MIG will listen to any export notifications and then send the data on based on the SCU value of the notification.

Reply:
I have updated MIG diagram to receive notifications from a pub-sub service and load output data directly from storage.